### ISSUE TITLE
unindent with `shift+tab` anywhere in a line 

### ISSUE THREAD

**Issue by georgexsh** [TablePlus/TablePlus#1193]
1. Which driver are you using and version of it (Ex: PostgreSQL 10.0):
mysql 5

2. Which TablePlus build number are you using (the number on the welcome screen, Ex: build 81):
226

3. The steps to reproduce this issue:

now, `shift+tab` only works if something in a line is selected, at least it could unindent if the cursor is at the first no space column of a line. if it works at any column is even better.

**Comment by jim256** [comment]
I agree that this would be really nice and is something I'm accustomed to.
